epoxy-processor/src/main/java/com/airbnb/epoxy/DataBindingAttributeInfo.kt

```diff
@@ -0,0 +1,26 @@
+package com.airbnb.epoxy
+
+import javax.lang.model.element.ExecutableElement
+import javax.lang.model.element.VariableElement
+
+import com.airbnb.epoxy.Utils.removeSetPrefix
+
+internal class DataBindingAttributeInfo(
```

Comment:
nit: I'd like to follow the conventions at https://kotlinlang.org/docs/reference/coding-conventions.html and have the closing parenthesis be on a new line